dashboard/app/controllers/aichat_controller.rb

```diff
@@ -29,7 +29,7 @@ def chat_completion
   private def get_response_body
     # Check for profanity
     locale = params[:locale] || "en"
-    filter_result = ShareFiltering.find_failure(params[:newMessage][:chatMessageText], locale)
+    filter_result = ShareFiltering.find_profanity_failure(params[:newMessage][:chatMessageText], locale)
```

Comment:
This method was added recently by https://github.com/code-dot-org/code-dot-org/pull/59414.